deployment/permissions/partner.0.ini

```diff
@@ -1350,3 +1350,9 @@ permission208.partnerId = 0
 permission208.type = 1
 permission208.name = INTEGRATION_ACTIONS 
 permission208.friendlyName = "integration_actions"
+
+permission209.partnerId = 0
+permission209.type = 1
+permission209.name = LIKE_LIST
```

Comment:
done. all other other remarks corrected as well.
it should be noted that admin users can assign non-admin-user roles with this permission.